hhs_oauth_server/settings/base.py

```diff
@@ -547,6 +547,8 @@
 
 SLSX_HEALTH_CHECK_ENDPOINT = env(
     'DJANGO_SLSX_HEALTH_CHECK_ENDPOINT', 'https://dev.accounts.cms.gov/health')
+MEDICARE_REDIRECT_URI_V2 = env(
+    'DJANGO_MEDICARE_REDIRECT_URI_v2', 'http://localhost:8000/mymedicare/sls-callback-v2')
```

Comment:
@JFU-GIT See previous comment related to the SLS/SLSx callback. I believe we need to use the same callback for the current SLS to work when deployed.

Reply:
Will take a look to see how to remember if we are in a v1 flow or v2, currently I use 2 mymedicare_cb endpoint urls to propagate the version info along.